Store/Get-LargeMailboxFolderStatistics.ps1

```diff
@@ -0,0 +1,68 @@
+﻿# Copyright (c) Microsoft Corporation.
```

Comment:
Let's avoid making a new folder structure here. Since this script is geared towards O365/M365 we should have it in that folder instead.

You can just place it at the root of that folder, and we will change the structure of that folder later. That should not be done within this PR.

Reply:
Yeah, we are trying to avoid adding too many components